go/cmd/gh-ost/main.go

```diff
@@ -149,6 +150,13 @@ func main() {
 	if migrationContext.SwitchToRowBinlogFormat && migrationContext.AssumeRBR {
 		log.Fatalf("--switch-to-rbr and --assume-rbr are mutually exclusive")
 	}
+	if *testOnReplicaWithManualReplicationControl {
+		if migrationContext.TestOnReplica {
+			log.Fatalf("--test-on-replica-manual-replication-control and --test-on-replica are mutually exclusive")
+		}
+		migrationContext.TestOnReplica = true
+		migrationContext.ManualReplicationControl = true
+	}
```

Comment:
@shlomi-noach , in my opinion the code reads more complicated to me when there are two flags that have the same core functionality, with this one slight variation.

I think the semantics would be more clear if we still used the `--test-on-replica` flag and had a separate flag that just customized this one aspect of its behavior.  So if one wants to have manual replication control, one would use the args:

```
--test-on-replica --manual-replication-control
```

Without the extra flag, `--test-on-replica` would behave as it always has.

What do you think?